com.mudeprolinux.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.mudeprolinux.whakarere
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: whakarere
+
+finish-args:
+  # X11 access
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to talk to the network:
+  - --share=network
+  # PulseAudio
+  - --socket=pulseaudio
+
+modules:
+  - name: Whakarere
+    buildsystem: simple
+    build-commands:
+      - install -D whakarere.py /app/bin/whakarere
+      - install -Dm644 whakarere.png /app/share/icons/hicolor/512x512/apps/com.tomudeprolinuxbagin.whakarere.png
+      - install -Dm644 whakarere.desktop /app/share/applications/com.mudeprolinux.whakarere.desktop
+    sources:
+      - type: file
+        path: whakarere.py
```

Comment:
where does it get this file?

Reply:
added the file again.

Reply:
no. This is not what I said either. 

Flathub is not meant to host the application.

Reply:
I see you mean where have I hosted the file correct? apologies for the noob question but it my first pr on anything. and thank you for your time.